ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -352,6 +353,10 @@ class SimpleOperationTracker implements OperationTracker {
     totalReplicaCount = replicaPool.size();
     originatingDcOfflineReplicaCount =
         getReplicasByState(originatingDcName, EnumSet.of(ReplicaState.OFFLINE)).values().size();
+    originatingDcEligibleReplicaCount = replicaPool.stream()
+        .filter(replicaId -> replicaId.getDataNodeId().getDatacenterName().equals(this.originatingDcName))
```

Comment:
When we generate SimpleOperationTracker, do we always provide "originatingDcName"?

Reply:
originatingDcEligibleReplicaCount is counting the replicas in the "eligible pool/replicaPool", the pool we uses to send requests.
    originatingDcTotalReplicaCount is counting the allReplicas which is the Helix ideal view. Basically it's always 3.